src/data/data.json

```diff
@@ -4,8 +4,8 @@
       "Commander": [
```

Comment:
Is it possible to import/use constants in a jsonfile? or do a need to make this a js file that returns a json object?

Reply:
also renamed this file to data for you @voiid

Reply:
😘

Not that I know of but haven't tried.

Reply:
@voiid Any thoughts of data best practices? Does it make sense to keep it all in one file, or split by faction or type?

Reply:
To be honest, I don't think it matters what the structure of the data is as long as you create a clear interface for accessing the data.

If you abstract away the structure of the data behind a utility that provides well-named helpers such as `getFaction` and `getAllFactions`, you'll be well covered for any changes you might need longer-term in data-structure. You'll also save yourself a lot of hassle everywhere you end up importing the data because you won't need to duplicate as much code to access specific factions.

The main consideration would probably be data transfer size. If you only ever render one faction at a time, transferring the whole file all the time is wasteful. Caveat being I don't expect this to be particularly big so yolo.

As long as your utility is the only thing handling interactions with this file, your bases are well covered for any structural changes you'll need longer-term since you would just have to update the implementations of those handful of functions. Avoid hasty re-structures or overcomplicated breakdowns of things until the requirements for the structure reveal themselves to you over time. Cover your ass to make change easy, but wait until the use cases present themselves to you rather than trying to plan in pure hypotheticals.